ui/src/actions/sampleview.js

```diff
@@ -41,8 +40,8 @@ export function setAperture(size) {
   return { type: 'SET_APERTURE', size };
 }
 
-export function setStepSize(name, value) {
-  return { type: 'SET_STEP_SIZE', componentName: 'sample_view', name, value };
+export function setMotorStep(role, value) {
+  return { type: 'SET_MOTOR_STEP', role, value };
```

Comment:
Making the action creator more specific, since we now use the `role` to find the `uiproperties` object to update, and this makes sense only for sample view motors.